hazelcast-jet-distribution/src/root/release_notes.txt

```diff
@@ -1,9 +1,75 @@
-In this Release Notes document, you will find the new features and fixes of the
-Hazelcast Jet release. The numbers in the square brackets refer to the issue number
-in Hazelcast Jet GitHub repository. You can reach to the full descriptions of the
-issues at https://github.com/hazelcast/hazelcast-jet/issues/<issue number>.
 
-The label in the brackets refer to the respective module.
+
+                    Hazelcast Jet Release Notes
+
+This document keeps the history of Hazelcast Jet releases. We try to
+enumerate all the non-trivial changes done to the product. The changes
+are catogerized into:
+
+    1. New features
+    2. Enhancements to the existing features
+    3. Fixes of bugs and other problems
+    4. Breaking changes: those that result either in compile errors or changes
```

Comment:
4. Breaking changes that result in ...

Reply:
That would change the meaning, though. I can put it it parentheses.